src/WebJobs.Extensions.OpenAI/EmbeddingsConverter.cs

```diff
@@ -33,7 +36,13 @@ public EmbeddingsConverter(OpenAIClient openAIClient, ILoggerFactory loggerFacto
         CancellationToken cancellationToken)
     {
         EmbeddingsContext response = await this.ConvertCoreAsync(input, cancellationToken);
-        return JsonConvert.SerializeObject(response);
+
+        // Note: we need this converter as Azure.AI.OpenAI does not support System.Text.Json serialization since their constructors are internal
+        var options = new JsonSerializerOptions
```

Comment:
Make this a `static readonly` field so that we don't need to allocate it every time we do a conversion.